### PR TITLE
Lets you fake wearing handcuffs by wielding them, automatically doing this on resist out

### PR DESCRIPTION
**Pull request by lewcc**
## What Does This PR Do
Lets you fake wearing handcuffs by wielding them. Also, if you resist out of handcuffs that don't break on removal (regular handcuffs, toy handcuffs, but not zipties), you'll also silently put them into your hands once you break out of them/
This fake handcuffing will look on examine to be a normal handcuffing, but it'll be immediately obvious that you're pulling a fast one under any scrutiny.

## Why It's Good For The Game
This changes how "silent" unbuckling works. If you're paying attention, you'll catch the chat message. Otherwise, if you're able to slip out of real handcuffs, you might get the chance to pull a fast one and pretend to be handcuffed.
This might be useful in other circumstances, too. You can put handcuffs on yourself and pretend to be the victim, for example, only to quickly take your cuffs off and apply them to whoever was unlucky enough to believe you!

## Images of changes

https://github.com/ParadiseSS13/Paradise/assets/89928798/4e0ec778-91c5-45e6-a674-602b8ae6dddb

image

## Testing
Handcuffed myself a bunch and resisted out, as well as just wielding the cuffs.

## Changelog
:cl:
tweak: You can now fake-wear handcuffs by using them in-hand.
tweak: Resisting out of reusable handcuffs will put the cuffs into your hand, and fake the handcuffing.
/:cl:

### CODE REVIEW

**Comment by Mitchs98** [comment]
The ability to fake handcuffs seems fairly cool and all. I really do not see the need to make uncuffing be safer with the addition of stealthily putting them in your hands and faking the uncuffing too. Frankly, I don't see the need nor benefit of simply stealthily putting them in your hands either. Getting yourself uncuffed is already a big enough benefit to getting yourself uncuffed, to be honest.

Now if you're able to fake the cuffs after having them drop off normally and produce a noticeable message? Great. That's fine. It being automatic and removing cuffs being stealthy really isn't to me. It's already easy enough to not notice someone removing cuffs as it is if there's a lot going on.

**Comment by AffectedArc07** [comment]
I feel like the fake cuffs and stealth unbuckle should be separate PRs, but that's Mr.

File: code/game/objects/items/weapons/handcuffs.dm [hunk 1]
```diff
@@ -25,6 +25,31 @@
...
+		unwield_callback = CALLBACK(src, PROC_REF(unwield)) \
+	)
+
+/obj/item/restraints/handcuffs/proc/wield(obj/item/source, mob/living/carbon/human/user)
+	if(!istype(user))
+		return
+	user.remove_overlay(HANDCUFF_LAYER)
+	user.overlays_standing[HANDCUFF_LAYER] = mutable_appearance('icons/mob/mob.dmi', cuffed_state, layer = -HANDCUFF_LAYER, color = color)
+	user.apply_overlay(HANDCUFF_LAYER)
+	item_state = "weijf2349-h238h9f23"  // thanks for the string contra
```
Comment (AffectedArc07) [inline code/game/objects/items/weapons/handcuffs.dm:42]: What on earth

Comment (lewcc) [reply code/game/objects/items/weapons/handcuffs.dm:42]: Setting the item state to a random string gives us an empty one, vs. just setting it to null which I think pulls the icon state name

**Comment by Komrad822** [comment]
Correct me if im wrong but you must first unbuckle before you can uncuff. This is the usual interaction between antag and sec in processing. Its always been on sec to pay attention to their chat. If they dont then that sucks for them imho. 

Where i see this having more of an impact is the interactions between antags and their victims. Antags dont always have the benefit of buckling their target as they have to stay on the move most of the time. If you as the victim are able to stealthily uncuff, you can wait for the right moment to act while the antag still thinks youre cuffed instead of trying to run away while still deep in maints of in a long hallway.

Also this allows for more victim rp as if youre an antag, you can fake being cuffed as sec arrives to the scene but be ready to scidaddle if they dont buy it or get the jump on the arriving sec so you catch them with their pants down since on initial inspection you look like a non threat.

File: code/game/objects/items/weapons/handcuffs.dm (hunk 1 above)
Comment (AffectedArc07) [reply code/game/objects/items/weapons/handcuffs.dm:42]: Surely setting the opacity to 101 or similar is a better solution?

Comment (lewcc) [reply code/game/objects/items/weapons/handcuffs.dm:42]: Can you do that for item states? It's specifically for the in-hand overlay

**Comment by ppi13** [comment]
I can see this is something that can have a hefty balance impact, and I would say this is something where having a test merge, during high pop and low pop, and gain some actual data on this.  
My immediate gut reaction was "No, this is not gonna be bad" but it could possible be good, and worst case scenario we figure out this is a terrible idea and end the TM early.
The interesting thing is that you cant just buckle cuff and mostly forget, coming back every minute to rebuckle, and you should be able to see the truth by checking their inventory, wow they are holding their cuffs in their hands, or you go down and see, wow they are not handcuffed, as well as trying to actually cuff em, or have a beepsky nearby,  so its not 100% without counterplay.
Like, I want this to be a possible thing, because its not just the station vs security, its also the station vs antags, and doing a sneaky uncufferoo, and you wall push em and run to freedom.

**Comment by Mitchs98** [comment]
> I can see this is something that can have a hefty balance impact, and I would say this is something where having a test merge, during high pop and low pop, and gain some actual data on this. My immediate gut reaction was "No, this is not gonna be bad" but it could possible be good, and worst case scenario we figure out this is a terrible idea and end the TM early. The interesting thing is that you cant just buckle cuff and mostly forget, coming back every minute to rebuckle, and you should be able to see the truth by checking their inventory, wow they are holding their cuffs in their hands, or you go down and see, wow they are not handcuffed, as well as trying to actually cuff em, or have a beepsky nearby, so its not 100% without counterplay. Like, I want this to be a possible thing, because its not just the station vs security, its also the station vs antags, and doing a sneaky uncufferoo, and you wall push em and run to freedom.

I'd definitely be willing to see a test merge, myself, to form a better opinion. And I do think this is something that needs a test merge to see how it plays out. As it is now I just see it being a major pain for security. But who knows, maybe it could make things a bit more interesting?

As said they do actually have to unbuckle still so that does help quite a bit in terms of balancing it.

**Comment by lewcc** [comment]
Doesn't seem to be too popular with the balance team, I think I'll shelve this one for now.

**Pull request closed without merging**